Alx-l/git-workshop
# [FEAT] add arrow

**QA**

1. Open index.html in your favorite browser.
2. [ ] You should see 2 items.
3. [ ] You can either, add new items, remove existing ones, complete existing ones
